### PR TITLE
use getClickedBlock instead of getTargetBlock

### PR DESCRIPTION
**Pull request by Mahagon**
This should fix Illegal state exception of block burn #585

### CODE REVIEW

File: src/main/java/de/diddiz/LogBlock/listeners/BlockBurnLogging.java [hunk 1]
```diff
@@ -31,12 +32,11 @@ public void onBlockBurn(BlockBurnEvent event) {
...
 	}
 	
-	
 	@EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
 	public void onExtinguish(PlayerInteractEvent event) {
-		Player player = event.getPlayer();
-		Block block = player.getTargetBlock(null, 5);
 		if(event.getAction().equals(Action.LEFT_CLICK_BLOCK)){
+			Player player = event.getPlayer();
+			Block block = event.getClickedBlock().getRelative(BlockFace.UP);
```
Comment (frymaster) [inline src/main/java/de/diddiz/LogBlock/listeners/BlockBurnLogging.java]: I think ``getClickedBlock()`` can be null

Comment (Mahagon) [inline src/main/java/de/diddiz/LogBlock/listeners/BlockBurnLogging.java]: are you sure? 
in a Action.LEFT_CLICK_BLOCK ?

:O i would wonder

**Comment by frymaster** [comment]
d'oh, I didn't see it was already inside an if...
going to test now

File: src/main/java/de/diddiz/LogBlock/listeners/BlockBurnLogging.java (hunk 1 above)
Comment (frymaster) [inline src/main/java/de/diddiz/LogBlock/listeners/BlockBurnLogging.java]: Is ``event.getBlockFace()`` useful rather than assuming up?

**Comment by Mahagon** [comment]
you are absolutely right. i burned down a lot but didn't thought about trees etc.... :D

**Comment by frymaster** [comment]
committed in 8c232b59970cc959a2a3fe10cf58f2e6ece86927 thanks :+1:

**Pull request closed without merging**